hbase-rest/src/test/java/org/apache/hadoop/hbase/rest/PerformanceEvaluation.java

```diff
@@ -708,33 +709,26 @@ private Path writeInputFile(final Configuration c) throws IOException {
     FileSystem fs = FileSystem.get(c);
     fs.mkdirs(inputDir);
     Path inputFile = new Path(inputDir, "input.txt");
-    PrintStream out = new PrintStream(fs.create(inputFile));
     // Make input random.
-    Map<Integer, String> m = new TreeMap<>();
-    Hash h = MurmurHash.getInstance();
-    int perClientRows = (this.R / this.N);
-    try {
+    try (PrintStream out = new PrintStream(fs.create(inputFile))) {
+      Map<Integer, String> m = new TreeMap<>();
+      Hash h = MurmurHash.getInstance();
+      int perClientRows = (this.R / this.N);
       for (int i = 0; i < 10; i++) {
         for (int j = 0; j < N; j++) {
-          String s = "tableName=" + this.tableName +
-              ", startRow=" + ((j * perClientRows) + (i * (perClientRows/10))) +
-              ", perClientRunRows=" + (perClientRows / 10) +
-              ", totalRows=" + this.R +
-              ", clients=" + this.N +
-              ", flushCommits=" + this.flushCommits +
-              ", writeToWAL=" + this.writeToWAL +
-              ", useTags=" + this.useTags +
-              ", noOfTags=" + this.noOfTags;
+          String s = "tableName=" + this.tableName + ", startRow=" + ((j * perClientRows) + (i * (
+            perClientRows / 10))) + ", perClientRunRows=" + (perClientRows / 10) + ", totalRows="
+            + this.R + ", clients=" + this.N + ", flushCommits=" + this.flushCommits
+            + ", writeToWAL=" + this.writeToWAL + ", useTags=" + this.useTags + ", noOfTags="
+            + this.noOfTags;
```

Comment:
Good to use StringBuilder may be?

Reply:
Updated the patch to use `StringBuilder` instead of `String` at this one.